manifests/pipeline.yaml

```diff
@@ -52,6 +52,8 @@ objects:
     kind: ImageStream
     metadata:
       name: jenkins-s2i
+      annotations:
+        coreos.com/deploy-default: true
```

Comment:
these annotations are causing me some grief on first startup:

```
Updating:                                                                                
Error from server (BadRequest): error when creating "STDIN": ImageStream in version "v1" cannot be handled as a ImageStream: v1.ImageStream.ObjectMeta: v1.ObjectMeta.Annotations: ReadString: expects " or n, but found t, error found
 in #10 byte of ...|default":true},"labe|..., bigger context ...|ata":{"annotations":{"coreos.com/deploy-default":true},"labels":{"app":"fedora-coreos","template":"f|...
Traceback (most recent call last):           
  File "./deploy", line 235, in <module>      
    sys.exit(main())                         
  File "./deploy", line 38, in main                                                        
    update_resources(args, resources)        
  File "./deploy", line 164, in update_resources                                                                                                                                                                                           check=True, stdout=subprocess.PIPE)      
  File "/usr/lib64/python3.7/subprocess.py", line 487, in run
    output=stdout, stderr=stderr)            
subprocess.CalledProcessError: Command '['oc', 'create', '--filename', '-']' returned non-zero exit status 1.

```

If I delete all the newly added `coreos.com/deploy-default: true` annotations and all `--all` to my `deploy` command then it succeeds.

Reply:
Hmm weird, I didn't hit this on my local server. But yeah, looks like it's complaining about string vs bool.